buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1523,7 +1523,14 @@ artifactId=${project.name}
 
       project.task('setupVirtualenv')  {
         doLast {
-          project.exec { commandLine 'virtualenv', "${project.ext.envdir}" }
+          def virtualenvCmd = [
+            'virtualenv',
+            "${project.ext.envdir}",
+          ]
+          if (project.hasProperty('python3')) {
+            virtualenvCmd += '--python=python3'
```

Comment:
In the future, I'd like us to be able to customize which Python 3 version to use (3.5, 3.6, etc). Dockerfile can take an argument for the FROM clause as well. This can happen in the future PR, but for the scope of this PR I'd like to make sure that we use the same Python 3 version to build the SDK as we use in the Dockerfile. So we I think should pass `--python=python3.5`, but we could make this a constant somewhere up in the file and make a note that the version should match the version we [inside the Docker container](https://github.com/apache/beam/blob/3a531323322b3f5bbd695b64744a0603a1c97607/sdks/python/container/py3/Dockerfile#L19).

Reply:
You are right. We should consider other version of python 3 in the future. Currently I don't have a good answer yet. So let's use `python3` in general and specify `--python=python35` (I think this is the right format) in virtualenv cmd. A constant with comments sounds good.